src/Forte.Optimizely.ContentUsage/Frontend/Styles/_variables.scss

```diff
@@ -1,5 +1,6 @@
 @import 'variables/breakpoints';
 
 :root {
```

Comment:
Isn't optimizely oui using sass variables for colors? Maybe we should do the same? See `node_modules\optimizely-oui\src\oui\_oui-variables.scss` for reference

Reply:
Also, maybe these colors are declared already? Please search in `@optimizely\design-tokens\dist\scss\_colors.scss`

Reply:
We should reuse as much as possible